src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div>
+    <p>
+      Add the people and corporations/firms who will have a role in your company. People can have multiple roles;
+      Corporation/firms can only be Incorporators.
+    </p>
+    Your application must include the following:
+    <ul>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.COMPLETING_PARTY, 1, 'EXACT')" class="cp-valid">mdi-check</v-icon>
```

Comment:
I'm wondering out loud if we should have a red X here if the completing party has been removed (or all incorporators or all directors)? Please follow up with Scott.

I don't think we should have a red X initially, but only if the respective person is removed.

Reply:
I checked this with Scott and even showed him how it is working. He mentioned that the red X should be shown only when you return to the page by clicking the link

Reply:
OK.